src/main/java/be/seeseemelk/mockbukkit/tags/internal/InternalTagParser.java

```diff
@@ -0,0 +1,91 @@
+package be.seeseemelk.mockbukkit.tags.internal;
+
+import be.seeseemelk.mockbukkit.MockBukkit;
+import be.seeseemelk.mockbukkit.tags.TagRegistry;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import org.bukkit.Material;
+import org.bukkit.NamespacedKey;
+import org.bukkit.Tag;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.EnumSet;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+public class InternalTagParser
+{
+	private static final Pattern MINECRAFT_MATERIAL = Pattern.compile("minecraft:[a-z0-9_]+");
+	private static final Pattern MINECRAFT_TAG = Pattern.compile("#minecraft:[a-z_]+");
+
+	public void insertInternalTagValues(InternalTagRegistry internalTagRegistry) throws IOException, InternalTagMisconfigurationException
+	{
+		String path = "/internal_tags/" + internalTagRegistry.name().toLowerCase() + "/";
+		for(InternalTag<?> internalTag : internalTagRegistry.getRelatedTags()){
+			String filePath = path + internalTag.getName().toLowerCase() + ".json";
+			System.out.println(filePath);
```

Comment:
```suggestion

```
I think a debug Statement slipped through

Reply:
HAHA! I missed that XD